save-core/src/commonMain/kotlin/org/cqfn/save/core/Save.kt

```diff
@@ -73,7 +62,11 @@ class Save(
                 reporter.beforeAll()
 
                 // discover plugins from the test configuration
-                discoverPluginsAndUpdateTestConfig(testConfig)
+                testConfig.pluginConfigs.clear()
```

Comment:
Did we not clear it before? Why is it needed now?

Reply:
Yes, It should be empty. Removed